docs/release-notes_exporter_v2.0.0-rc.1.md

```diff
@@ -0,0 +1,255 @@
+# SSV Exporter v2 — Pre‑Release Notes (v2.0.0‑rc.1)
+
+> **Audience:** SSV node operators and clients integrating with the exporter API.  
+> **Focus:** configuration (files/flags/env), API surface (HTTP + WebSocket), compatibility, and upgrade steps.  
+> **Compatibility:** v2 keeps legacy clients working. 
+>     - In *standard* mode, the legacy `/v1/exporter/decideds` response keeps the v1 format and error semantics (fail‑fast on processing errors). Error message strings may differ from exporter v1. 
+>     - In *archive* mode, the same path is backed by the new v2 pipeline and returns a retro‑compatible `data` entry plus an additional `errors` array; processing is **best‑effort** and returns all valid results when possible. If a request yields **no valid results** and **at least one meaningful error**, the API returns an HTTP error.
+
+---
+
+## Overview
+
+Exporter v2 introduces **full duty tracing** (consensus rounds, pre/post partial signatures), **committee‑level views**, richer filtering, OpenAPI definitions, and a clearer configuration model with **two operating modes**:
+
+- **archive** — full traces persisted to disk, new `/traces/*` endpoints, and `/decideds` served by the v2 pipeline (adds `errors` array).
+- **standard** — v1‑compatible participants only; the legacy `/decideds` payload is preserved and old data is pruned automatically.
+
+v2 uses **best‑effort responses**: it returns partial results alongside non‑fatal issues in an `errors` array. If a request yields **no valid data** and **at least one meaningful error**, it returns an HTTP error instead of an empty success.
+
+---
+
+## Quick Start
+
+### 1) Choose a mode and enable the exporter
+
+```yaml
+# config.yaml (excerpt)
+exporter:
+  Enabled: true              # enable exporter pipeline
+  Mode: archive              # "archive" | "standard"
+  RetainSlots: 50400         # used in standard mode only (~7 days at 12s/slot)
+```
+
+**Archive mode requirements & effects**
+- Persists detailed traces on disk (Pebble). Higher CPU/disk and P2P traffic (subscribes to all subnets).
```

Comment:
pebble db is now used for exporter in all modes. might worth a separate section.